Yatzy-Kata/Yatzy.Core/Data/Category.cs

```diff
@@ -0,0 +1,98 @@
+namespace Yatzy_Kata.Data;
+
+public abstract record Category()
+{
+    public abstract int CalculateScore(List<int> dice);
+}
+
+public sealed record Aces() : Category
+{
+    public override int CalculateScore(List<int> dice) => dice.Where(ones => ones == 1).Sum();
+}
+
+public sealed record Twos() : Category
+{
+    public override int CalculateScore(List<int> dice) => dice.Where(twos => twos == 2).Sum();
+}
+
+public sealed record Threes() : Category
+{
+    public override int CalculateScore(List<int> dice) => dice.Where(threes => threes == 3).Sum();
+}
+
+public sealed record Fours() : Category
+{
+    public override int CalculateScore(List<int> dice) => dice.Where(fours => fours == 4).Sum();
+}
+
+public sealed record Fives() : Category
```

Comment:
Great use of C#!!

Reply:
Thanks, although I chose to store the categories encapsulated with their calculation logic this way, I think there probably was a better way to go about this. But I wanted to try unfamiliar things.